.github/workflows/example-images-vulnerability-scanning.yaml

```diff
@@ -0,0 +1,17 @@
+name: Published images vulnerability scanning
```

Comment:
nit: Should the name for this workflow be different in order to more easily tell it apart when looking at all the (action) jobs.

Also, a shorter name would be nice.

Reply:
Yes. It's supposed to be different.